mlos_bench/mlos_bench/storage/sql/experiment_data.py

```diff
@@ -34,6 +38,55 @@ def __init__(self, *, engine: Engine, schema: DbSchema, exp_id: str,
         self._engine = engine
         self._schema = schema
 
+    @property
+    def objectives(self) -> Dict[str, str]:
+        objectives: Dict[str, str] = {}
+        # First try to lookup the objectives from the experiment metadata in the storage layer.
+        if hasattr(self._schema, "objectives"):
+            with self._engine.connect() as conn:
+                objectives_db_data = conn.execute(
+                    self._schema.objectives.select().where(
+                        self._schema.objectives.c.exp_id == self._exp_id,
+                    ).order_by(
+                        self._schema.objectives.c.optimization_target.asc(),
```

Comment:
I wonder if the order of optimization targets is important? Maybe we need an extra column like "order" or "priority" in the `objectives` table?

Reply:
In true multi-objective, no, they should be equally explored by the optimizer, but it's not a bad thing to include for future support.
How about just an optional floating point "weight"?

Reply:
I added the column for now, but am leaving it not fully supported in the rest of the APIs just yet as we don't fully support multi-objective yet anyways and adding weights would change a lot of these APIs more that I need for the initial dabl one sooner.